docs/metrics/queue.md

```diff
@@ -0,0 +1,63 @@
+---
+id: queue
+title: Queue
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Queue
+### Application
+Eech queue has a `<queue_name> queue_app` metric to trace the applications in the queue.
+`<queue_name> queue_app` metrics records the number of applications in different states.
+These application states include `running`, `accepted`, `rejected`, `failed` and `completed`. 
+`<queue_name> queue_app` metrics record container states including `released`, `allocated`. 
```

Comment:
We need to add a blank line above to allow the "Metric Type" to wrap to the next line.
![image](https://github.com/apache/yunikorn-site/assets/48400525/847839fa-73e5-495d-a01c-59a45b3c05bf)